rust/operator-binary/src/main.rs

```diff
@@ -58,21 +60,54 @@ async fn main() -> anyhow::Result<()> {
         Command::Run(ProductOperatorRun {
             product_config,
             watch_namespace,
-            tracing_target,
+            telemetry_arguments,
             cluster_info_opts,
         }) => {
-            stackable_operator::logging::initialize_logging(
-                "AIRFLOW_OPERATOR_LOG",
-                APP_NAME,
-                tracing_target,
-            );
-            stackable_operator::utils::print_startup_string(
-                crate_description!(),
-                crate_version!(),
-                built_info::GIT_VERSION,
-                built_info::TARGET,
-                built_info::BUILT_TIME_UTC,
-                built_info::RUSTC_VERSION,
+            let _tracing_guard = Tracing::builder()
+                .service_name("airflow-operator")
+                .with_console_output((
+                    "AIRFLOW_OPERATOR_LOG",
+                    LevelFilter::INFO,
+                    !telemetry_arguments.no_console_output,
+                ))
+                // note, before, log dir was set via an env: `AIRFLOW_OPERATOR_LOG_DIRECTORY`.
+                // See: https://github.com/stackabletech/operator-rs/blob/f035997fca85a54238c8de895389cc50b4d421e2/crates/stackable-operator/src/logging/mod.rs#L40
+                // Now it will be `ROLLING_LOGS` (or via `--rolling-logs <DIRECTORY>`)
```

Comment:
Replace with:

```suggestion
                // NOTE (@NickLarsenNZ): Before stackable-telemetry was used, the log directory was
                // set via an env: `AIRFLOW_OPERATOR_LOG_DIRECTORY`.
                // See: https://github.com/stackabletech/operator-rs/blob/f035997fca85a54238c8de895389cc50b4d421e2/crates/stackable-operator/src/logging/mod.rs#L40
                // Now it will be `ROLLING_LOGS` (or via `--rolling-logs <DIRECTORY>`).
```

Reply:
This is also a **BREAKING** change, but I don't expect many users to be affected.

We should still see how it gets set in the chart.

Reply:
We should overide the `ROLLING_LOGS` env var to be `ROLLING_LOGS_DIR` (the argument can stay as is though).